FacialRecog.py

```diff
@@ -126,14 +132,15 @@ def get_images_and_encodings(frame) -> Tuple[list, list]:
 
 """
 given a list of encodings from frame and a list of all
-system encdoings, returns a list of lists of boolean values
+system encodings, returns a list of lists of boolean values
 """
 def compare_encodings(frame_encodings, all_encodings):
     if len(frame_encodings) == 0:
         return None
 
     encodings = []
-    for face in all_encodings: 
-        encodings.append(face_recognition.compare_faces(face, frame_encodings))
+    for face in all_encodings:
+        for entry in frame_encodings:
```

Comment:
what's the purpose of adding the inside loop? .compare_faces can compare a face against a list of faces, unless this is just for readability

Reply:
I think that it was left over from initially trying to make all the functions handle multiple cameras instead of just the recog script. will push what i am working with rn